File: program/find-volume-of-a-sphere/find_volume_of_a_sphere.go
package main
 
import (
   "fmt"
   "math"
)
 
func volume (radius float64) float64 {
   return 4.0/3.0 * math.Pi * math.Pow(radius,3)
}
 
func main() {
  var radius float64
  fmt.Println("Enter value of radius:")
  fmt.Scanf(&radius)
  fmt.Println("Volume of Sphere is:",volume(radius))
}

File: program/find-volume-of-a-sphere/FindVolumeOfASphere.rb
def volumeOfSphere(radius)
  4/3 * 3.14 * radius * radius * radius
end

if $PROGRAM_NAME == __FILE__
  print 'Please enter the radius : '
  radius = gets.chomp.to_i

  puts "Volume of Sphere :  #{volumeOfSphere(radius)}"
end
